### PR TITLE
Debug test on Linux

### PR DESCRIPTION
**Pull request by 0101**
These are the symbol uses I see:

```fsharp
[|TestProject_88c93ff2, Binding, (1,7--1,27);
  generic parameter a, UseInType, (2,15--2,17);
  generic parameter a, UseInType, (2,31--2,33);
  generic parameter a, UseInType, (2,31--2,33); TFirst, Binding, (2,21--2,27);
  TFirstV_1`1, Binding, (2,5--2,14); val x, Binding, (3,6--3,7);
  TFirst, Use, (4,4--4,10); val x, Use, (4,11--4,12); val f, Binding, (3,4--3,5);
  val x, Binding, (5,7--5,8); val op_Addition, Use, (5,13--5,14);
  val x, Use, (5,11--5,12); val f2, Binding, (5,4--5,6);
  ModuleFirst, Binding, (1,28--1,39); TestProject_88c93ff2, Open, (1,7--1,27);
  generic parameter a, UseInType, (2,16--2,18);
  generic parameter a, UseInType, (2,33--2,35);
  generic parameter a, UseInType, (2,33--2,35); TSecond, Binding, (2,22--2,29);
  TSecondV_1`1, Binding, (2,5--2,15); val x, Binding, (3,6--3,7);
  ModuleFirst, Use, (4,4--4,15); val f, Use, (4,4--4,17);
  val x, Use, (4,18--4,19); TSecond, Use, (5,4--5,11); val x, Use, (5,12--5,13);
  val f, Binding, (3,4--3,5); val x, Binding, (6,7--6,8);
  val op_Addition, Use, (6,13--6,14); val x, Use, (6,11--6,12);
  val f2, Binding, (6,4--6,6); ModuleSecond, Binding, (1,28--1,40);
  TestProject_88c93ff2, Open, (1,7--1,27);
  generic parameter a, UseInType, (2,15--2,17);
  generic parameter a, UseInType, (2,31--2,33);
  generic parameter a, UseInType, (2,31--2,33); TThird, Binding, (2,21--2,27);
  TThirdV_1`1, Binding, (2,5--2,14); val x, Binding, (3,6--3,7);
  ModuleFirst, Use, (4,4--4,15); val f, Use, (4,4--4,17);
  val x, Use, (4,18--4,19); TThird, Use, (5,4--5,10); val x, Use, (5,11--5,12);
  val f, Binding, (3,4--3,5); val x, Binding, (6,7--6,8);
  val op_Addition, Use, (6,13--6,14); val x, Use, (6,11--6,12);
  val f2, Binding, (6,4--6,6); ModuleThird, Binding, (1,28--1,39);
  TestProject_88c93ff2, Open, (1,7--1,27);
  generic parameter a, UseInType, (2,14--2,16);
  generic parameter a, UseInType, (2,29--2,31);
  generic parameter a, UseInType, (2,29--2,31); TLast, Binding, (2,20--2,25);
  TLastV_1`1, Binding, (2,5--2,13); val x, Binding, (3,6--3,7);
  ModuleSecond, Use, (4,4--4,16); val f, Use, (4,4--4,18);
  val x, Use, (4,19--4,20); ModuleThird, Use, (5,4--5,15);
  val f, Use, (5,4--5,17); val x, Use, (5,18--5,19); TLast, Use, (6,4--6,9);
  val x, Use, (6,10--6,11); val f, Binding, (3,4--3,5);
  val x, Binding, (7,7--7,8); val op_Addition, Use, (7,13--7,14);
  val x, Use, (7,11--7,12); val f2, Binding, (7,4--7,6);
  EntryPointAttribute, UseInAttribute, (9,2--9,12);
  EntryPointAttribute, UseInType, (9,2--9,12); member .ctor, Use, (9,2--9,12);
  val op_PipeRight, Use, (11,7--11,9); val f, Use, (11,3--11,4);
  val ignore, Use, (11,10--11,16); val printfn, Use, (12,3--12,10);
  val main, Binding, (10,4--10,8); ModuleLast, Binding, (1,28--1,38)|]
```

### CODE REVIEW

**Comment by vzarytovskii** [comment]
I don't get why is it flaky and sometimes shows different amount of results.

**Comment by 0101** [comment]
> I don't get why is it flaky and sometimes shows different amount of results.

Is it still? I think we should just merge this so we can at least see what's missing (or extra) when it happens.

**Comment by vzarytovskii** [comment]
> > I don't get why is it flaky and sometimes shows different amount of results.
> 
> 
> 
> Is it still? I think we should just merge this so we can at least see what's missing (or extra) when it happens. 

Surprisingly it just blow ups out of the blue sometimes. And then it's ok for two weeks...

File: tests/FSharp.Compiler.ComponentTests/FSharpChecker/CommonWorkflows.fs [hunk 1]
```diff
@@ -150,19 +150,19 @@ let GetAllUsesOfAllSymbols() =
...
+            let! checkProjectResults = checker.ParseAndCheckProject(options)
             return checkProjectResults.GetAllUsesOfAllSymbols()
         } |> Async.RunSynchronously
 
 
     traceProvider.ForceFlush() |> ignore
     traceProvider.Dispose()
 
-    Assert.Equal(79, result.Length)
+    if result.Length <> 79 then failwith $"Expected 80 symbolUses, got {result.Length}:\n%A{result}"
```
Comment (T-Gro) [inline tests/FSharp.Compiler.ComponentTests/FSharpChecker/CommonWorkflows.fs:168]:
```suggestion
    if result.Length <> 79 then failwith $"Expected 79 symbolUses, got {result.Length}:\n%A{result}"
```

**Review by T-Gro — APPROVED** [review]

**Comment by vzarytovskii** [comment]
Closing it, feel free to reopen

**Comment by 0101** [comment]
I still think we should merge this to see what happens when the test fails.

**Pull request merged**